programs/clearing_house/src/lib.rs

```diff
@@ -929,6 +950,298 @@ pub mod clearing_house {
         Ok(())
     }
 
+    pub fn add_insurance_liquidity<'info>(
```

Comment:
maybe we rename the shares to ip_shares (insurance liquidity) ?

Reply:
renamed to use "stake"